sdk/communication/communication-email/samples-dev/sendEmailWithInlineAttachments.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Sends an email with an inline image attachment
+ */
+
+import { EmailClient, EmailMessage } from "@azure/communication-email";
+
+// Load the .env file (you will need to set these environment variables)
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["COMMUNICATION_CONNECTION_STRING"] || "";
+const senderAddress = process.env["SENDER_ADDRESS"] || "";
+const recipientAddress = process.env["RECIPIENT_ADDRESS"] || "";
+
+const sendEmailWithAttachments = async (): Promise<void> => {
+  // Create the Email Client
+  const emailClient: EmailClient = new EmailClient(connectionString);
+
+  // Create the Email Message to be sent
+  const message: EmailMessage = {
+    senderAddress: senderAddress,
+    content: {
+      subject: "This is the subject",
+      plainText: "This is the body",
+      html: '<html><h1>This is the body<img src="cid:inline_image" /></h1></html>',
```

Comment:
nit: It feels a bit odd to have the img inside the h1 (and also a text that says "body" but is a headline). Maybe let's remove the h1 tags? (I didn't really pay attention to that on the other PRs)

```suggestion
      html: '<html>This is the body<img src="cid:inline_image" /></html>',
```